### PR TITLE
Add more LTS version to CI matrix

### PR DESCRIPTION
**Pull request by SergeAstapov**
another piece of overdue maintenance

### CODE REVIEW

File: packages/ember-engines/config/ember-try.js [hunk 1]
```diff
@@ -45,6 +45,17 @@ module.exports = async function () {
...
         }
       },
+      {
+        name: "ember-lts-4.12",
+        npm: {
+          devDependencies: {
+            "ember-source": "~4.8.0",
+            "ember-cli": "~3.28.0",
+            "ember-cli-app-version": "^5.0.0",
+            "@ember/legacy-built-in-components": "~0.4.0",
```
Comment (void-mAlex) [inline packages/ember-engines/config/ember-try.js:55]: starting from this version we really should use latest legacy built in components 0.5.x which has a peer on this ember source

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -54,6 +54,9 @@ jobs:
           - ember-lts-3.24
           - ember-lts-3.28
           - ember-lts-4.4
+          - ember-lts-4.8
```
Comment (void-mAlex) [inline .github/workflows/ci.yml:57]: not sure what the value of keeping 4.4 around is

Comment (SergeAstapov) [reply .github/workflows/ci.yml:57]: my thought process is that it should be each LTS from oldest supported

File: packages/ember-engines/config/ember-try.js (hunk 1 above)
Comment (SergeAstapov) [reply packages/ember-engines/config/ember-try.js:55]: good point! Thanks!

File: .github/workflows/ci.yml (hunk 2 above)
Comment (void-mAlex) [reply .github/workflows/ci.yml:57]: ci isn't that slow now and this only adds some extra work which is what I'm concerned about, but it's probably fine

Comment (villander) [reply .github/workflows/ci.yml:57]: I prefer adding the latest embroider to it
```suggestion
          - ember-lts-4.12
          - ember-lts-5.4
          - ember-release
          - ember-beta
          - ember-canary
          - embroider-safe
          - embroider-optimized
```

**Review by villander — CHANGES_REQUESTED** [review]
We need to make it more realistic with ember status quo

File: .github/workflows/ci.yml (hunk 2 above)
Comment (SergeAstapov) [reply .github/workflows/ci.yml:57]: @villander v5 and embroider are omit intentionally, until we get #855 over the finish line

Comment (villander) [reply .github/workflows/ci.yml:57]: @SergeAstapov can we get the finish line before change the pipeline? I’m pretty sure it has more priority than internal things

Comment (void-mAlex) [reply .github/workflows/ci.yml:57]: 855 adds the extra pipelines you're talking about @villander, this bit is fine as is

**Comment by void-mAlex** [comment]
this pr is ready, we're just holding off till release plan is set up so we benefit from automation for it

File: .github/workflows/ci.yml (hunk 2 above)
Comment (villander) [reply .github/workflows/ci.yml:57]: @void-mAlex thanks for the clarifications

**Comment by SergeAstapov** [comment]
as #872 landed, merging this to test out release-plan cc @void-mAlex

**Pull request merged**